src/main/java/com/denizenscript/depenizen/bukkit/properties/towny/TownyChatPlayerProperties.java

```diff
@@ -54,16 +53,8 @@ private TownyChatPlayerProperties(PlayerTag player) {
 
     PlayerTag player;
 
-    @Override
-    public String getAttribute(Attribute attribute) {
-        if (attribute == null) {
-            return null;
-        }
-
-        if (attribute.startsWith("townychat")) {
-            Chat plugin = (Chat) TownyChatBridge.instance.plugin;
-            ChannelsHolder holder = plugin.getChannelsHandler();
-            attribute = attribute.fulfill(1);
+    public static void registerTags() {
+        PropertyParser.<TownyChatPlayerProperties>registerTag("townychat", (attribute, object) -> {
```

Comment:
This wasn't tested, was it?
None of these tags will work.

Reply:
Nothing Towny wise was tested yet as I hadn't finished it haha